AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1018,6 +1034,14 @@ - (BOOL)openURL:(NSURL *)url {
 
       // Update distribution group ID which is added to logs.
       [self.distributeInfoTracker updateDistributionGroupId:queryDistributionGroupId];
+        
+      // Only if we have managed to retrieve the Distribution group ID we should update the distribution session count.
+      latestSessionId = [MSSessionContext sessionIdAt:[NSDate date]];
+      
+      // If Analytics SDK is disabled session Id is null and there is no neeed to updte the distribution session count.
+      if(latestSessionId) {
```

Comment:
space between `if` and `(`.
Please run formatter. There is .clang-format file in the repo that you can use.